components/engine_traits/src/engines.rs

```diff
@@ -11,7 +13,7 @@ pub struct KvEngines<K, R> {
     pub shared_block_cache: bool,
 }
 
-impl<K: KvEngine, R: KvEngine> KvEngines<K, R> {
+impl<K: KvEngine, R: RaftEngine> KvEngines<K, R> {
```

Comment:
`KvEngines` is kind of confusing after adding `RaftEngine`. Maybe rename to `Engines` or others